src/main/java/seedu/address/logic/commands/RemoveFromConsultCommand.java

```diff
@@ -0,0 +1,187 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_CONSULTATIONS;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.consultation.Consultation;
+import seedu.address.model.consultation.exceptions.PersonNotFoundInConsultException;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+/**
+ * Removes a person identified using his/her name from the specified consultation by index.
+ */
+public class RemoveFromConsultCommand extends Command {
+    public static final String COMMAND_WORD = "removefromconsult";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Removes a person identified using his/her name from the specified consultation "
+            + "by index in the displayed consultation list.\n"
```

Comment:
Not sure if its just me or if the sentence is a bit odd/unclear. Think "Removes a person identified using his/her name from a consultation. The consultation is specified by index..." works is clearer and sounds better.

Reply:
Good suggestion!